.github/workflows/CI.yaml

```diff
@@ -37,49 +37,40 @@ jobs:
     - name: Create Psi4 Environment
       uses: conda-incubator/setup-miniconda@v2
       with:
-        activate-environment: p4dev
+        activate-environment: p4env
         environment-file: devtools/conda-envs/psi.yaml
         python-version: ${{ matrix.python-version }}
         auto-activate-base: true
 
-    - name: Environment Information
+    - name: Install Psi4
       run: |
         conda info
         conda list --show-channel-urls
         hash -r
         env
-
-    - name: Install Psi4
-      run: |
-        conda info
-        conda list --show-channel-urls
-        conda install psi4 python=${{ matrix.python-version }} -c conda-forge 
+        conda install psi4 python=${{ matrix.python-version }} -c conda-forge
         ls -l $CONDA
 
     - name: Test Psi4 Python Loading
       run: |
         export PATH=$CONDA_PREFIX/bin:$PATH
         export PYTHONPATH=$CONDA_PREFIX/lib:$PYTHONPATH
+        conda env list
         python -c 'import psi4'
 
     - name: Install PyCC and Deps
       run: |
-        env
-        export PATH=$CONDA_PREFIX/bin:$PATH
-        export PYTHONPATH=$CONDA_PREFIX/lib:$PYTHONPATH
         python -m pip install .
-        python -m pip install opt_einsum
```

Comment:
this makes it so the CI is depending on `setup.py` to handle the installation of `opt_einsum`, `scipy`, and `torch`, correct? this is much better!

Reply:
Yep!